src/components/UtilizationBar/UtilizationBar.js

```diff
@@ -0,0 +1,145 @@
+import { ProgressBar } from '../ProgressBar';
+import ClassNames from 'classnames';
+import PropTypes from 'prop-types';
+import { OverlayTrigger } from '../OverlayTrigger';
+import { Tooltip } from '../Tooltip';
+import React from 'react';
+
+const AvailableTooltipFunction = (max, now) => {
+  return <Tooltip id={Math.random()}>Available {max - now} %</Tooltip>;
+};
+
+const UsedTooltipFunction = now => {
+  return <Tooltip id={Math.random()}>Used {now} %</Tooltip>;
+};
+
+const UtilizationBar = props => {
+  const barStyle = () => {
+    if (props.thresholdWarning && props.thresholdError) {
+      let style = 'success';
+      if (props.thresholdWarning <= props.now) {
+        style = 'warning';
+      }
+      if (props.thresholdError <= props.now) {
+        style = 'danger';
+      }
+      return style;
+    } else {
+      return null;
+    }
+  };
+
+  const labelClasses = top => {
+    return ClassNames(
+      { 'progress-label-top-right': top, 'progress-label-right': !top },
+      'pull-right'
+    );
+  };
+
+  const mainDivClasses = (onSide, mainDivClasses) => {
+    return ClassNames({
+      'progress-container': onSide && props.description,
+      'progress-description-left': onSide && props.description,
+      'progress-label-right': onSide && props.label,
+      mainDivClasses: mainDivClasses
+    });
+  };
+
+  return (
+    <div
+      className={mainDivClasses(
+        !props.descriptionPlacementTop,
+        props.mainDivClasses
+      )}
+    >
+      <div className={props.descriptionPlacementTop ? null : 'progress-bar'}>
+        {props.label && (
+          <span
+            className={labelClasses(props.descriptionPlacementTop)}
+            style={{ display: 'inline-block', fontSize: '12px' }}
+          >
+            <strong style={{ fontWeight: 600 }}>
```

Comment:
isn't <strong> enough? why change the font-weight?

Reply:
For **consistency**. In original [example](https://rawgit.com/patternfly/patternfly/master-dist/dist/tests/cards.html#) it's `font-weight: 600` and without it there's difference between label on top of bar and one next to it.
<img width="1155" alt="screen shot 2018-02-13 at 9 32 11 am" src="https://user-images.githubusercontent.com/9210860/36140330-3272cdce-10a1-11e8-8984-8f81d5e6bf24.png">
So is it ok as long as it goes to css? :)

Reply:
Shouldn't be necessary as it should use the same html structure as patternfly to use the patternfly styling. See the original example you linked above for the patternfly html structure.